packages/serverless/src/awslambda.ts

```diff
@@ -78,6 +78,9 @@ export function init(options: Sentry.NodeOptions = {}): void {
     version: Sentry.SDK_VERSION,
   };
 
+  // Point the SDK to the Lambda Extension instead of the host specified in the DSN
+  options.dsn = extensionRelayDSN(options.dsn);
```

Comment:
This will break people who use `Sentry.AWSLambda` without having a lambda layer installed.

@antonpirker do you know how we could fix this? My first instinct would be to guard this line behind an init option.

Reply:
@antonpirker came up with the idea to guard this behind a flag that is only set when initializing the SDK from within the Sentry Lambda Layer: dab5f85